client/packages/requisitions/src/ResponseRequisition/ListView/ListView.tsx

```diff
@@ -133,6 +133,8 @@ export const ResponseRequisitionListView: FC = () => {
   }
   columnDefinitions.push(['comment', { minWidth: 400, Cell: TooltipTextCell }]);
```

Comment:
Can we make the comment section a bit smaller so that all the columns fit onto the screen~?

```suggestion
  columnDefinitions.push(['comment', { minWidth: 350, Cell: TooltipTextCell }]);
```